plugins/destination/firehose/client/client_test.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+)
+
+const streamARN = "cq-playground-test"
+
+func TestPluginJSON(t *testing.T) {
+	p := destination.NewPlugin("firehose", "development", New, destination.WithDefaultBatchSize(500))
+	destination.PluginTestSuiteRunner(t, p,
+		Spec{
+			NoRotate:  true,
+			StreamARN: streamARN,
+		},
+		destination.PluginTestSuiteTests{
+			SkipOverwrite:     true,
+			SkipDeleteStale:   true,
+			SkipSecondAppend:  true,
+			SkipMigrateAppend: true,
+			SkipAppend:        true,
```

Comment:
I think this is skipping.... everything? Except `MigrateOverwrite`, but I'm not sure why 😄

Reply:
Because I didn't know that one existed... Will add it